mobile/src/components/OfflineIndicator.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ReactElement, useEffect, useContext } from "react";
+import {
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  Image,
+  ViewStyle,
+} from "react-native";
+import { TextStyles } from "../styles/TextStyles";
+import { Colors } from "../styles/Colors";
+import NetInfo from "@react-native-community/netinfo";
+import { I18nContext } from "../context/I18nContext";
+
+type OfflineIndicatorProps = {
+  children: ReactElement;
+  variant?: "orange" | "white";
+  style?: ViewStyle;
+};
```

Comment:
Minor thing - can you go back through this file and replace all the double quotes with single quotes for consistency? I think the linter broke over the summer :| I'll try to get that fixed so this is less of a hassle to keep up with